dashboard/test/controllers/home_controller_test.rb

```diff
@@ -324,7 +324,7 @@ def setup_user_with_gallery
 
   test 'workshop organizers who are regional partner program managers see application dashboard links' do
     sign_in create(:workshop_organizer, :as_regional_partner_program_manager, :with_terms_of_service)
-    assert_queries 10 do
+    assert_queries 12 do
```

Comment:
This can be 11. See above.

Reply:
I'm not able to find the above you're referring to; when can it be 11?

Reply:
https://github.com/code-dot-org/code-dot-org/pull/19481#discussion_r155097994

Reply:
Oh, hah! I thought you meant above in the test file. Thanks!

Reply:
Heh, that was pretty ambiguous, especially after you made the update and that comment disappeared ;)